src/FakeItEasy.Analyzer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Reflection;

```

Comment:
Most of this file should come from CommonAssemblyInfo.cs.

Note that that CommonAssemblyInfo.cs has too much in it right now, e.g. the `SuppressMessage` and `InternalsVisibleTo` bits. It should be trimmed down to whatever is common across both the FakeItEasy and FakeItEasy.Analyzer packages and the rest should sit in local AssemblyInfo.cs files for the specific projects (FakeItEasy and FakeItEasy.Net35 would share a single AssemblyInfo.cs).